bodhi/server/models.py

```diff
@@ -2748,6 +2748,9 @@ def comment(self, session, text, karma=0, author=None, anonymous=False,
         """
         if not author:
             raise ValueError('You must provide a comment author')
+            
+        if not text:
+            raise ValueError('You must provide text in comment')
```

Comment:
As @mattiaverga commented in #2009, we don't necessarily want to restrict users from providing only karma. I suggest changing this to ```if not text and not karma```.